src/components/pages/Offers/Offer/Stocks/__specs__/Stocks.spec.jsx

```diff
@@ -14,6 +14,9 @@ import { getToday } from 'utils/date'
 import { bulkFakeApiCreateOrEditStock, loadFakeApiOffer, loadFakeApiStocks } from 'utils/fakeApi'
 import { queryByTextTrimHtml } from 'utils/testHelpers'
 
+const GUYANA_CAYENNE_DEPT = '973'
```

Comment:
```suggestion
const CAYENNE_GUYANA_DEPT = '973'
```
Pour coller à ce que tu as fait pour la constante en dessous : VILLE_PAYS_DEPT